PyPortal_Tides/admiralty_tides.py

```diff
@@ -0,0 +1,105 @@
+import time
+import board
+from adafruit_pyportal import PyPortal
+from adafruit_bitmap_font import bitmap_font
+from adafruit_display_text.label import Label
+import json
+from secrets import secrets
```

Comment:
Here's a boiler plate that other programs have used for the secrets.py import:
```python
# Get wifi details and more from a secrets.py file
try:
    from secrets import secrets
except ImportError:
    print("WiFi secrets are kept in secrets.py, please add them there!")
raise
```

Example:
https://github.com/adafruit/Adafruit_Learning_System_Guides/blob/master/PyPortal_OpenWeather/openweather.py#L14

Reply:
changes made